src/WCSimStackingAction.cc

```diff
@@ -23,13 +23,13 @@ G4ClassificationOfNewTrack WCSimStackingAction::ClassifyNewTrack
   G4String WCIDCollectionName = DetConstruct->GetIDCollectionName();
   G4ClassificationOfNewTrack classification    = fWaiting;
   G4ParticleDefinition*      particleType      = aTrack->GetDefinition();
-  
+  G4float correctionfactor = 1/0.73; //Collection factor for 20inchPMT CE setting
```

Comment:
The comment on the correctionfactor should be clearer so that we remember why we did this. Something like "Correction factor added in July 2015 to account for the measured relative pe number differences between R3600 and box and line PDs."